src/exportable/styles.css

```
@@ -0,0 +1,80 @@
+/*
+ * Self-contained styles for @flexprice/pricing-ui.
+ *
+ * Consumers import this once: `import '@flexprice/pricing-ui/style.css'`.
+ * It ships the theme tokens (light + dark) the components rely on, plus only the Tailwind
+ * utilities actually used by the pricing components (scoped via tailwind.pricing.config.js).
+ *
+ * Theme it by overriding these CSS variables on a wrapping element, e.g.:
+ *   .my-scope { --fp-primary: 243 75% 59%; }
+ */
+@tailwind base;
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)